api/tools/build.gradle

```diff
@@ -0,0 +1,56 @@
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'org.springframework.boot'
+
+def db_host = System.getenv("DB_HOST")
+def db_port = System.getenv("DB_PORT")
+def workbench_db_user = System.getenv("WORKBENCH_DB_USER")
+def workbench_db_password = System.getenv("WORKBENCH_DB_PASSWORD")
+
+task loadConfig(type: JavaExec) {
+    classpath sourceSets.main.runtimeClasspath
+    main = "org.pmiops.workbench.tools.ConfigLoader"
+    systemProperties = [
+      'spring.datasource.driver-class-name': 'com.mysql.jdbc.Driver',
+      'spring.datasource.url': 'jdbc:mysql://${db_host}:${db_port}/workbench',
+      'spring.datasource.username': '${workbench_db_user}',
+      'spring.datasource.password': '${workbench_db_password}'
+    ]
+    args config_file
```

Comment:
This is coming from the gradle properties file? How does it get changed for another environment?

Reply:
You can pass -pconfig_file=<file> on the command line and it will override gradle.properties.